### DISCUSSION TITLE
Hero 11 mini LOGB=400 question

### DISCUSSION THREAD

**Discussion by Hydrostreamin** [gopro/labs#576] [General]
When I apply the LOGB QR code, it applies it to all of the color profiles instead of just flat. Is that what is supposed to happen? Or am I missing something?  I’ve tried mL in the additional command section as well.

**Comment by dnewman-gpsw** [comment]
mL was added for HERO12 only to enable GP-Log (which very like much oMLOGB=400.)

Most Labs commands apply globally, so all modes are effected by any oMwxyz=? command.  This is a tad inelegant, but it vastly less complex to implement, and Labs is about testing ideas, not a polished implementation (that is for the main FW.)

**Comment by Hydrostreamin** [comment]
Ahh, that makes sense. I’m pretty new to using labs, Thanks for the reply!
